src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,168 @@
+//            } else if (firstWord.equals("delete")) {
+//                if (s.length <= 1) { throw new EmptyArgumentException();}
+//                int num = Integer.parseInt(s[1]) - 1;
+//                Task t = arrL.remove(num);
+//                System.out.println("Noted. I've removed this task:\n" + t +
+//                        "\nNow you have " + arrL.size() + " tasks in the list.");
+//            } else if (firstWord.equals("todo") || firstWord.equals("deadline") || firstWord.equals("event")){
+//                if (s.length <= 1) { throw new EmptyArgumentException();}
+//                if (firstWord.equals("todo")) {
+//                    arrL.add(new ToDo(s[1], false));
+//                }
+//                else if (firstWord.equals("deadline")) {
+//                    String st[] = s[1].split(" /by ", 2);
+//                    arrL.add(new Deadline(st[0], false, LocalDate.parse(st[1])));
+//                } else if (firstWord.equals("event")) {
+//                    String st[] = s[1].split(" /from ", 2);
+//                    String stt[] = st[1].split(" /to ", 2);
+//                    arrL.add(new Event(st[0], false, stt[0], stt[1]));
+//                }
+//                System.out.println("Got it. I've added this task:\n" + arrL.get(arrL.size()-1)
+//                    + "\nNow you have " + arrL.size() + " tasks in the list.");
+//            } else {
+//                throw new InvalidCommandException();
+//            }
+//        }
+//
+//    }
+//
+//
+//
+//    public static void populateArray(List<Task> l, String filePath) throws FileNotFoundException {
+//        File f = new File(filePath); // create a File for the given file path
+//        try {
+//            if(!f.exists()) {
+//                f.createNewFile();
+//            }
+//        } catch (IOException e) {
+//            // TODO Auto-generated catch block
+//            e.printStackTrace();
+//        }
+//
+//        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+//
+//        while (s.hasNext()) {
+//            String str = s.nextLine();
+//            String parts[] = str.split("~", 5);
+//
+//            switch (parts[0]) {
+//                case "T":
+//                    l.add(new ToDo(parts[2], parts[1].equals("1")));
+//                    break;
+//                case "D":
+//                    l.add(new Deadline(parts[2], parts[1].equals("1"), LocalDate.parse(parts[3])));
+//                    break;
+//                case "E":
+//                    l.add(new Event(parts[2], parts[1].equals("1"), parts[3], parts[4]));
+//                    break;
+//            }
+//        }
+//    }
+//
+//
+//    public static void save(List<Task> l, String filePath) throws IOException {
+//        FileWriter fw = new FileWriter(filePath);
+//        for (Task i : l) {
+//            fw.write(i.saveFormat() + "\n");
+//        }
+//        fw.close();
+//    }
+//}
```

Comment:
May want to consider removing this chunk of comments if there is no need for it anymore.